src/CPE/cpelang.c

```diff
@@ -55,53 +55,6 @@ void cpe_lang_model_export(const struct cpe_lang_model *spec, const char *file)
 	cpe_lang_model_export_xml(spec, file);
 }
 
-/*
- * Add new platform entry to @a platformspec
- * @note @a platformspec will take over memory management of @a platform
- * @param platformspec list of platforms being extended
- * @param platform platform to add to the list
- * @return true on success
- */
-static bool cpe_language_match_expr(struct cpe_name **cpe, size_t n, const struct cpe_testexpr *expr)
```

Comment:
ACK and completely removed.

https://github.com/search?q=language%3Ac+cpe_language_match_expr&type=Code